packages/api-genesis/src/example_queries/keyNetworkInfoByronShelley.graphql

```diff
@@ -0,0 +1,29 @@
+query keyNetworkInfo {
```

Comment:
There are values like epochLength and slotLength which are Protocol agnostic, so could be added to the Byron object too